Symfony/CS/Fixer.php

```diff
@@ -210,6 +212,12 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
                 FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_INVALID)
             );
 
+            $this->errorsManager->report(new Error(
+                Error::TYPE_LINTING,
+                $this->getFileRelativePathname($file),
+                $e->getMessage()
```

Comment:
This will return text grabbed from `php -l foo.php` process, not so user-friendly...